packages/doctor/lib/node-detector.js

```diff
@@ -31,67 +29,10 @@ class NodeDetector {
     return nodePath;
   }
 
-  static async retrieveUsingAppleScript() {
-    if (!system.isMac()) {
-      log.debug('Not on Darwin, skipping Apple Script');
-      return null;
-    }
-
-    const appScript = [
-      'try',
-      '  set appiumIsRunning to false',
-      '  tell application "System Events"',
-      '    set appiumIsRunning to name of every process contains "Appium"',
-      '  end tell',
-      '  if appiumIsRunning then',
-      '    tell application "Appium" to return node path',
-      '  end if',
-      'end try',
-      'return "NULL"',
-    ].join('\n');
-    let stdout;
-    try {
-      stdout = (await exec('osascript', ['-e', appScript])).stdout;
-    } catch (err) {
-      log.debug(err);
-      return null;
-    }
-    let nodePath = stdout.replace('\n', '');
-    if (await fs.exists(nodePath)) {
-      log.debug(`Node binary found using AppleScript at: ${nodePath}`);
-      return nodePath;
-    } else {
-      log.debug('Node binary not found using AppleScript.');
-      return null;
-    }
-  }
-
-  static async retrieveUsingAppiumConfigFile() {
-    let jsonobj;
-    try {
-      const appiumConfigPath = path.resolve(__dirname, '..', '..', '.appiumconfig.json');
-      if (await fs.exists(appiumConfigPath)) {
-        jsonobj = JSON.parse(await fs.readFile(appiumConfigPath, 'utf8'));
-      }
-    } catch (err) {
-      log.debug(err);
-      return null;
-    }
-    if (jsonobj && jsonobj.node_bin && (await fs.exists(jsonobj.node_bin))) {
-      log.debug(`Node binary found using .appiumconfig.json at: ${jsonobj.node_bin}`);
-      return jsonobj.node_bin;
-    } else {
-      log.debug('Node binary not found in the .appiumconfig.json file.');
-      return null;
-    }
-  }
-
   static async detect() {
     let nodePath =
       (await NodeDetector.retrieveUsingSystemCall()) ||
-      (await NodeDetector.retrieveInCommonPlaces()) ||
-      (await NodeDetector.retrieveUsingAppleScript()) ||
-      (await NodeDetector.retrieveUsingAppiumConfigFile());
+      (await NodeDetector.retrieveInCommonPlaces());
```

Comment:
I would say this particular detection could be removed as well because Appium would anyway fail if such common folders are not part of PATH